_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
+layout: article
+section: On-prem
+category: component
```

Comment:
Why do we need this?

Reply:
Well, that's why. 
The original issue description according to your requirements -- http://prntscr.com/mki5aa

Reply:
Please read my requirements one more time and compare with your results please.

Reply:
That is the thing. I did exactly what you have said (according to the given requirements) from the very beginning, and the article won't appear just right where we need it. 
So referring back to the "Why do we need this?" question. We need this to make the article appear in the necessary section. If there is some other way to fix it, the way I didn't notice, please advice. I'd appreciate it.